docs/pages/choose-an-edition/teleport-enterprise/gcp-kms.mdx

```diff
@@ -8,6 +8,18 @@ This guide will show you how to set up your Teleport Cluster to use the Google
 Cloud Key Management Service (KMS) to store and handle the CA private key
 material used to sign all certificates issued by your Teleport cluster.
 
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["enterprise"]}
+  scopeOnly={true}
```

Comment:
Since the default scope is `oss`, I think there's a chance that a lot of users will miss this warning.

Reply:
Because this page is [only scoped](https://github.com/gravitational/teleport/blob/master/docs/config.json#L116) for enterprise, a user's `scope` key in the URL is always reset to Enterprise when you navigate to it. You can test this yourself in the browser.